2019-pydays-vienna/typos-correction-with-lookout.md

```diff
@@ -0,0 +1,36 @@
+# Title of the Talk
+
+**Updates**:  https://github.com/src-d/conferences/issues/133
+
+**Sent on**:  -
+
+**Status**:   on preparation
+
+**Author**:   Irina
+
+**Bio**: Junior Machine Learning Engineer at source{d}. Persue Master's degree of Computer Science and Data Analysis at MIPT (Moscow
+Institute of Physics and Technology). Study at Yandex School of Data Analysis.
+
+**Slides**:   -
+
+**Abstract**:
+
+We all make typos in code. Some are filtered at an early stage with help of IDEs or during compilation and tests.
```

Comment:
This is definitely too long. A good abstract is, say, 200 characters. If they ask for a longer description additionally to the abstract, you can use it there.